src/components/Coverage/Coverage.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SerialCoverage from './SerialCoverage';
+import MonographCoverage from './MonographCoverage';
+import { resourceTypes } from '../../constants';
+
+export default class Coverage extends React.Component {
+  static propTypes = {
+    pci: PropTypes.shape({
+      pti: PropTypes.shape({
+        titleInstance: PropTypes.shape({
+          dateMonographPublished: PropTypes.string,
+          monographVolume: PropTypes.string,
+          monographEdition: PropTypes.string,
+          name: PropTypes.string
+        }),
+      })
+    }),
+    line: PropTypes.shape({
+      resource: PropTypes.shape({
+        _object: PropTypes.shape({
+          pci: PropTypes.shape({
+            pti: PropTypes.shape({
+              titleInstance: PropTypes.shape({
+                dateMonographPublished: PropTypes.string,
+                monographVolume: PropTypes.string,
+                monographEdition: PropTypes.string,
+                name: PropTypes.string
+              })
+            })
+          })
+        })
+      })
+    }),
+    coveredEResource: PropTypes.shape({
```

Comment:
Can you change this prop to just `resource` or `eresource`? A covered eresource is something very specific to agreements (it's an eresource we've been granted via agreement lines), and we're using the term for things like basket contents where it doesn't apply.

Reply:
Have done :)